src/mocks/data.ts

```diff
@@ -722,6 +723,54 @@ export const accessRequests = [
   },
 ];
 
+export const accessGrants: AccessGrant[] = [
+  {
+    id: 'grant-ghga-8c4b9d5a1f0a',
+    user_id: 'doe@test.dev',
+    dataset_id: 'GHGAD12345678901234',
+    created: '2025-07-20T10:00:00Z',
+    valid_from: '2025-08-01T00:00:00Z',
+    valid_until: '2026-08-01T00:00:00Z',
+    user_name: 'John Doe',
+    user_title: 'Dr.',
+    user_email: 'doe@home.org',
+    dataset_title: 'Test dataset for details',
+    dac_alias: 'SOME-DAC',
+    dac_email: 'dac-main@some.org',
+    iva_id: '32b50c92-489f-4418-ace8-e7552e3cf36d',
+  },
+  {
+    id: 'grant-ghga-8c4b9d5a1f0b',
+    user_id: 'doe@test.dev',
+    dataset_id: 'GHGAD12345678901234',
```

Comment:
```suggestion
    dataset_id: 'GHGAD12345678901235',
```
so that we can test filtering and sorting

Reply:
done